junit_viewer.css

```diff
@@ -1,6 +1,6 @@
 body {
```

Comment:
there have been some changes to the css could you update this code and compare how it looks to double check everything is OK and update if needed